### PR TITLE
Add sample bibtex citation to README

### PR DESCRIPTION
**Pull request by wkalt**
Adds a suggested bibtex citation to the README.

### CODE REVIEW

**Comment by defunctzombie** [comment]
Seems good to me. Does this mirror the format other projects that want to be cited propose in their readme?

**Comment by wkalt** [comment]
ok I fixed that little snag. Here is the new screenshot.

Selection_006

**Comment by wkalt** [comment]
@defunctzombie I mostly stole this idea from rerun.io's readme, so the format matches theirs.

**Comment by wkalt** [comment]
Selection_007

File: README.md [hunk 1]
```diff
@@ -93,3 +93,26 @@ Tag a release matching the version number `releases/swift/vX.Y.Z`
...
 2. Tag a release matching the version number `releases/rust/vX.Y.Z`
+
+## Citations
+
+If you use MCAP in your research please cite it in your work. Our suggested
+citation format is:
+
+```
+@software{MCAP,
+title = {MCAP: serialization-agnostic log container file format},
```
Comment (jtbandes) [inline README.md:104]: indentation?

File: README.md [hunk 2]
```diff
@@ -93,3 +93,26 @@ Tag a release matching the version number `releases/swift/vX.Y.Z`
...
+author = {{Foxglove Developers}},
+url = {https://mcap.dev},
+version = {your version},
+date = {your date of access},
+year = {2024},
+publisher = {{Foxglove Technologies}},
+address = {Online},
```
Comment (jtbandes) [inline README.md:111]: are all of these fields needed?  `address = {Online}` doesn't seem like it is adding much value

File: README.md [hunk 3]
```diff
@@ -93,3 +93,26 @@ Tag a release matching the version number `releases/swift/vX.Y.Z`
...
+note = {Available from https://github.com/foxglove/mcap}
+}
+```
+
+Please replace the version and date files with the version of the software you
```
Comment (jtbandes) [inline README.md:116]:
```suggestion
Please replace the version and date fields with the version of the software you
```

File: README.md [hunk 4]
```diff
@@ -93,3 +93,26 @@ Tag a release matching the version number `releases/swift/vX.Y.Z`
 
 1. Update the version in rust/Cargo.toml
 2. Tag a release matching the version number `releases/rust/vX.Y.Z`
+
+## Citations
+
+If you use MCAP in your research please cite it in your work. Our suggested
```
Comment (jtbandes) [inline README.md:99]:
```suggestion
If you use MCAP in your research, please cite it in your work. Our suggested
```

**Comment by wkalt** [comment]
@jtbandes thanks, updated

**Review by jtbandes — APPROVED** [review]

**Pull request merged**